packages/webapp/src/hooks/api/useMutations.js

```diff
@@ -0,0 +1,43 @@
+/*
+ *  Copyright 2024 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+const useMutations = (namedMutations) => {
+  const results = namedMutations.map((mutation) => mutation.hook());
+  const data = results.map((result) => result[1]);
```

Comment:
Yeah good call, I am thinking that the scenario @SayakaOno mentioned about isSuccess and isError is probably quite common.  Wondering if the main usefulness of this hook is the structured/named response to avoid scenarios like `deleteAnimalsTrigger` ,`isDeletingAnimalsSuccess`, `isDeletingAnimalsError` etc in favour of `mutations['delAnimals'].lifecycle.isSuccess`. While not necessarily shorter it's more iterable I think.

Are you good if I keep only results and mutations and delete the rest? I could also possibly scrap this utility function.